### PR TITLE
Sept.25 Issues

### PR DESCRIPTION
**Pull request by mperrien**

### CODE REVIEW

**Comment by antoviaque** [comment]
@mperrien Thanks! Besides https://github.com/rue89-tech/edx-theme/issues/6 , is this ready on your side, we can start a review this week?

**Comment by mperrien** [comment]
Hello @antoviaque It's OK for me, you can start the review.

File: src/sass/pages/_dashboard.scss [hunk 1]
```diff
@@ -50,19 +53,26 @@
...
 									}
 									.course-image {
 									    height: auto;
 									    width: auto;
 									    min-height: 100%;
-										max-height: unset;
-									    min-width: 100%;
+										max-height: unset;//
+//									    min-width: 100%;
+//										max-width: 100%;
```
Comment (mtyaka) [inline src/sass/pages/_dashboard.scss]: @mperrien nit: can you remove the commented-out code?

File: src/sass/pages/_dashboard.scss [hunk 2]
```diff
@@ -126,8 +138,9 @@
 								.wrapper-course-actions {
 									width: 100%;
 									float: none;
-									position: absolute;
-									bottom: 20px;
+//									position: absolute;
+//									bottom: 20px;
```
Comment (mtyaka) [inline src/sass/pages/_dashboard.scss]: @mperrien nit: remove the commented code

**Comment by mtyaka** [comment]
@mperrien I deployed the updates to http://stage.mooc.rue89.com and verified the changes. It all looks good, there's only two things I noticed.

1) The subtitle of the selected unit is still black, not gray. Not sure if that's intentional or not.

screen shot 2015-10-01 at 16 52 49

2) The courses on the dashboard are not as high as they used to be. This might be intentional, please ignore my comment if it was changed like that on purpose.

Before:

screen shot 2015-10-01 at 16 59 14

After:

screen shot 2015-10-01 at 16 59 06

**Comment by mperrien** [comment]
Hello @mtyaka.
Thank you for your comments.
The commits above should fix these issues and a couple of others.
Let me know if you need / notice anything else.

**Comment by mtyaka** [comment]
Thanks @mperrien, the updates look good :+1:

**Pull request merged**